README.md

```diff
@@ -173,6 +173,30 @@ are detailed below:
   If you are an author of an original template not affiliated with any
   organization, only the standard package naming guidelines apply to you.
 
+- **Description:** Write simple, easily-understandable and succinct
+	descriptions.
+	- Keep it short. Try to maximise the content:length ratio and weigh your words
+		thoughtfully. No more than 100 characters, preferably much less.
+	- Avoid the word ‘Typst’, which is redundant unless your package or template
+		actually has to do with Typst itself or its ecosystem (like in the case of
+		[Mantys](https://typst.app/universe/package/mantys/) or
+		[t4t](https://typst.app/universe/package/t4t)).
+	- Don’t terminate your description with a full stop.
+	- Avoid the words ‘package’ for packages and ‘template’ for templates;
+		instead:
+		- Packages allow the user to *do* things; use the imperative mood. For
+			example, `Draw Venn diagrams` is better than `A package for drawing Venn
+			diagrams`.
+		- Templates allow the user to write certain *types* of documents; clearly
+			indicate the type of document your template allows. For example, `Master’s
+			thesis at the Unseen University` is better than `A template for writing a
+			master’s thesis at the Unseen University`. Omit the indefinite article at
+			the beginning of the description for conciseness.
+	- Bilingual descriptions are welcome. If you wish to describe your package in a
```

Comment:
Could please you remove this guideline?

Reply:
For context: We think for this to be truly useful, the README would also need to be localized, and we'd also want a more machine-readable format (like a per-language table). But that's more complexity than we want to deal with for now, so we feel like it's best to leave it out for now.